bundles/org.eclipse.ui.workbench.texteditor/src/org/eclipse/ui/texteditor/FindReplaceDialog.java

```diff
@@ -311,7 +314,6 @@ public void widgetSelected(SelectionEvent e) {
 						writeSelection();
 						updateButtonState(!somethingFound);
 						updateFindHistory();
-						evaluateFindReplaceStatus();
```

Comment:
Why remove this call (and the others like it in other listeners)?

Reply:
Thanks for pointing that out! I made an error earlier. The way I changed it, after selecting "Find" (if the regex option is not selected), nothing happens, even though it should display "String not found." I've fixed that now, but I'd like to hear your thoughts, @fedejeanne @Wittmaxi. Should the error message remain the same as before, or should we style it like we did with the regex search? Alternatively, we could handle it like in the find/replace overlay, where the text turns red as soon as the word isn't found.

before: 
![image](https://github.com/user-attachments/assets/2219ff68-07fd-45cb-88ab-9fbab33adfdf)

Reply:
>  Should the error message remain the same as before, or should we style it like we did with the regex search? Alternatively, we could handle it like in the find/replace overlay, where the text turns red as soon as the word isn't found.

Preserve the functionality as it is right now and leave that improvement for another PR. 

... small steps :-)

Reply:
done